_config.yml

```diff
@@ -112,7 +112,7 @@ defaults:
 
 # Just the Docs
 
-remote_theme: pmarsceill/just-the-docs
+theme: just-the-docs
```

Comment:
Somehow it worked for me locally with just `remote_them`, but actually including it in the Gemfile is probably safer (because it locks the version), more friendly to other developers (if you had problems running locally, someone else will too) and less dependent on GitHub.